elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/AggregationDataStore.java

```diff
@@ -30,12 +35,15 @@ public AggregationDataStore(QueryEngineFactory queryEngineFactory) {
      */
     @Override
     public void populateEntityDictionary(EntityDictionary dictionary) {
-        queryEngine = queryEngineFactory.buildQueryEngine(dictionary);
+        metaDataStore.loadMetaData(dictionary);
```

Comment:
This method (populateEntityDictionary) should actually bind entities into the dictionary.

Reply:
That happens when metadatastore populate the entity dictionary